source/channels/channels-settings.rst

```diff
@@ -206,6 +206,14 @@ Select which language Mattermost displays in the user interface. Options include
 - 中文 (繁體) - Traditional Chinese
 - 日本語 - Japanese
 
+Advanced settings
+~~~~~~~~~~~~~~~~~
+
+Using the Mattermost mobile app, you can delete local Mattermost files from your device. Access **Settings** by tapping on your profile picture.
```

Comment:
Maybe makes sense to clarify that the only stuff deleted will be the stuff regarding the actual server, not regarding every single server registered on the app.

Reply:
@larkox - I've drafted an update to clarify what data is/isn't deleted with this action. Anything you'd like to see changed?